operatorcourier/cli.py

```diff
@@ -0,0 +1,63 @@
+from operatorcourier import courier_api
+import argparse
+import sys
+
+def main():
+    """Generate the CLI bits
+    """
+    _CliParser()
```

Comment:
This looks weird to me. Object init should not do any exec operations, it should just create object.

```python
parser = _CliParser()  # just prepares parser
args = parser.parse()  # process args, can do some post parse checks

if args.command == 'push':
    do_push(args.x, args.y)
elif args.command == 'build':
   do_build(args.x, args.z)
else:
  raise RuntimeError('Unexpected commnad')
```

Reply:
Addressed in latest commit.